### ISSUE TITLE
Bruh, I found a vulnerability in GrassBandits.

### ISSUE THREAD

**Issue by Hardwaregore** [TechDudie/educaty#1]
@TechDudie 

If you type 
```
<script>window.location.replace("https://www.youtube.com/watch?v=dQw4w9WgXcQ");</script>
```

Into the feedback form and click submit, you can no longer access the reviews or staff page. FIX IT NOW!!!! You can see the result by visiting https://techdude.pythonanywhere.com and clicking reviews.

**Comment by Hardwaregore** [comment]
@TechDudie, your website is valuable to XSS (cross-site scripting) or javascript injection. Please fix this vulnerability before the site goes live at `https://thegrassbandist.tk`

**Comment by Hardwaregore** [comment]
@TechDudie, u know clearing the js cache wouldn't fix the vulnerability LOL

**Comment by TechDudie** [comment]
okay ill add some python to sanitize user input
if youre gonna do xss then ill whack the code

**Comment by Hardwaregore** [comment]
@TechDudie, after u done, create a snyk account. U can sign up here: https://app.snyk.io/share/8e3e8b1b-5adc-4f88-ade5-2e40c3561101. Snyk is a tool that scans ur code for vulnerabilitys. BRUH

**Comment by TechDudie** [comment]
Now that im replying to all my emails okay FINE I WILL FIX IT WITH HTML INPUT SANITIZATION BUT IM BUSY RIGHT NOW OKAY BRUH BE PATINET IDC THAT I SPELLED IT WRONG OKAY there ive said all my points I will fix it with html input sanitization but it will be a while

And about the thegrassbandits.tk……… I talked to siralexbigbrain YOU LIED
And STOP SPAMMING RANDOM REPOISTORIES I CREATE ITS ANNOYING PLEASE

@Hardwaregore

From: Hardwaregore ***@***.***>
Date: Monday, August 15, 2022 at 6:23 PM
To: TechDudie/educaty ***@***.***>
Cc: TechnoDotΩ ***@***.***>, Mention ***@***.***>
Subject: Re: [TechDudie/educaty] Bruh, I found a vulnerability in GrassBandits. (Issue #1)

Closed #1<https://github.com/TechDudie/educaty/issues/1> as completed.
